test/Amadevus.RecordGenerator.Test/RecordConstructorTests.cs

```diff
@@ -9,7 +9,7 @@ public class RecordConstructorTests : RecordTestsBase
         [Fact]
         public void Ctor_HasParametersNamedLikeProperties()
         {
-            new Item(Id: ItemId, Name: ItemName);
+            new Item(id: ItemId, name: ItemName);
```

Comment:
While at it, could you add tests that check camelCase names for all generated methods types as well?

Reply:
I've added tests and fixed the issue in 8ac523cfd654538a700d8e4d987470746f127dcb by using `value` as the parameter name for the all `With*` methods. I don't find there's any value in naming them uniquely based on the property name.